live-build/auto/config

```diff
@@ -31,24 +31,6 @@
 # delphix-platform.
 #
 lb config noauto \
```

Comment:
How does this `lb config` step interact with the `lb config` done in `scripts/run-live-build.sh`?

Reply:
The `auto` script here is basically a wrapper around the real `lb config` command, and `run-live-build.sh` is invoking this script. The invocation here provides the default options, which are added to / overridden by the options passed in from `run-live-build.sh`. This has some details on how it works: https://live-team.pages.debian.net/live-manual/html/live-manual/managing-a-configuration.en.html